.travis.yml

```diff
@@ -47,20 +47,29 @@ jobs:
       - *no_optimize
       - *no_imports
 
-    - env: &py36_env
+    - env:
       - PYTHON=3.6
+      - NUMPY=1.14.0
+      - PANDAS=0.22.0
+      - *no_coverage
+      - *no_optimize
+      - *imports
+
+    - env: &py36_dev
       - UPSTREAM_DEV=1  # Install nightly versions of NumPy, pandas, pyarrow
       - NUMPY=1.13.0    # these are overridden later
       - PANDAS=0.20.3
       - *no_coverage
       - *no_optimize
       - *imports
+      if: type != pull_request
 
-    - env: *py36_env
+    - env: *py36_dev
```

Comment:
I think we should use released versions on the osx tests instead. This used to be true, and then it wasn't when the dev tests were added. Ideally each split would test a different issue - the osx tests are for anything oddly osx specific. Adding in dev dependencies conflates things.

My preference would be the osx env matches the env you just added (with numpy=1.14.0).

Reply:
Agreed. Have switched macOS to the latest stable releases as suggested.